inc/cachify.class.php

```diff
@@ -1018,6 +1019,17 @@ private static function _cache_expires() {
 	}
 
 	/**
+	 * Determine if cache details should be printed in signature
+	 *
+	 * @since   2.3.0
+	 *
+	 * @return  bool  Show details in signature.
+	 */
+	private static function _signature_details() {
+		return isset(self::$options['sig_detail']) ? self::$options['sig_detail'] == 1 : false;
```

Comment:
I think, this could be a bit more readable. Isn't `isset(self::$options['sig_detail'])` always `true`?